c.php

```diff
@@ -312,21 +305,23 @@ function parse_function($lexer) {
 }
 
 $file_path = $argv[1];
-$source = file_get_contents($file_path);
+$source    = file_get_contents($file_path);
+
 if (!$source) exit(69);
+
 $lexer = new Lexer($file_path, $source);
-$func = parse_function($lexer);
+$func  = parse_function($lexer);
 if (!$func) exit(69);
 
-function literal_to_py($value) {
+function literal_to_py(string|int $value) : string {
     if (is_string($value)) {
         return "\"" . $value . "\"";
     } else {
         return (string)$value;
     }
 }
 
-foreach($func->body as &$stmt) {
```

Comment:
PHP does copy on write. The use of the reference operator *(`&`)* is neither necessary, nor recommended, as it guarantees a copy internally in this case, so it would be bad for performance.